Health/ViewModels/Cells/PersonalAlanSummaryCellViewModel.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  PersonalAlanSummaryCellViewModel.swift
+//  Health
+//
+//  Created by juks86 on 8/18/25.
+//
+
+import UIKit
+import Combine
+import HealthKit
+
+final class AIMonthlySummaryCellViewModel {
+    @Injected(.healthService) private var healthService: HealthService
+    @Injected private var promptBuilderService: (any PromptBuilderService)
+    @Injected private var userService: (any CoreDataUserService)
+    
+    private var alanService = AlanViewModel()
+    private var healthDataViewModel = HealthDataViewModel()
+    
+    private var cancellables = Set<AnyCancellable>()
+    private static var todayCache: String?
+    private static var cacheDate: String?
+    private(set) var itemID: ItemID
+    
+    // 셀 식별자
+    struct ItemID: Hashable {
+        let id: UUID = UUID()
+    }
+    
+    // 월간 요약 내용
+    struct Content: Hashable, Equatable {
+        let message: String
+    }
+    // 로딩 상태 관리
+    let stateSubject = CurrentValueSubject<LoadState<Content>, Never>(.idle)
+    
+    // 상태 변화 퍼블리셔
+    var statePublisher: AnyPublisher<LoadState<Content>, Never> {
+        stateSubject.eraseToAnyPublisher()
+    }
+    
+    // 상태 변화 콜백
+    var didChange: ((ItemID) -> Void)?
+    
+    // 초기화
+    init(itemID: ItemID) {
+        self.itemID = itemID
+        
+        // 포그라운드 진입 시 권한 재체크
+        NotificationCenter.default.publisher(for: UIApplication.willEnterForegroundNotification)
+            .sink { [weak self] _ in
+                Task {
+                    await self?.loadMonthlySummary()
+                }
+            }
+            .store(in: &cancellables)
+        
+        // 날짜 변경 감지 설정 (자정 넘어갈 때)
+        NotificationCenter.default.publisher(for: .NSCalendarDayChanged)
+            .sink { [weak self] _ in
+                Task {
+                    await self?.loadMonthlySummary()
+                }
+            }
+            .store(in: &cancellables)
+    }
+    
+    // 상태 업데이트
+    func setState(_ new: LoadState<Content>) {
+        stateSubject.send(new)
+        didChange?(itemID)
+    }
+}
+
+// MARK: - 월간 요약 로딩 로직
+extension AIMonthlySummaryCellViewModel {
+    
+    /// 월간 요약을 로딩합니다 (HealthDataViewModel 재사용)
+    @MainActor
+    func loadMonthlySummary() async {
+        setState(.loading)
+        
+        // 권한 체크
+        let hasStepPermission = await healthService.checkHasReadPermission(for: .stepCount)
+        let hasDistancePermission = await healthService.checkHasReadPermission(for: .distanceWalkingRunning)
+        let hasCaloriesPermission = await healthService.checkHasReadPermission(for: .activeEnergyBurned)
```

Comment:
병렬로 처리하면 조금 더 빠르긴 할거에요
```suggestion
        async let stepPermission = healthService.checkHasReadPermission(for: .stepCount)
        async let distancePermission = healthService.checkHasReadPermission(for: .distanceWalkingRunning)
        async let caloriesPermission = healthService.checkHasReadPermission(for: .activeEnergyBurned)
        
        let results = await (stepPermission, distancePermission, caloriesPermission)
        
        guard results.0 && results.1 && results.2 else {
            setState(.denied)
            return
        }
```